connectors/agent/component.py

```diff
@@ -0,0 +1,69 @@
+import sys
+
+from elastic_agent_client.client import V2Options, VersionInfo
+from elastic_agent_client.reader import new_v2_from_reader
+from elastic_agent_client.service.actions import ActionsService
+from elastic_agent_client.service.checkin import CheckinV2Service
+
+from connectors.agent.config import ConnectorsAgentConfigurationWrapper
+from connectors.agent.protocol import ConnectorActionHandler, ConnectorCheckinHandler
+from connectors.agent.service_manager import ConnectorServiceManager
+from connectors.services.base import MultiService
```

Comment:
This is using the MulitService from Connectors instead of from the elastic_agent_client. Which I _think_ is ok (and probably for the best) but just wanted to flag in case it wasn't intentional. I hadn't tested using ActionsService or CheckinV2Service with the Connectors MultiService.

Reply:
It's technically same code - we need to clean up the python-agent repo from some classes like this or move them to `example` namespace to not distribute these classes.